client/src/routes/WorkspaceRoute/Workspace.js

```diff
@@ -65,9 +66,60 @@ const move = (source, destination, droppableSource, droppableDestination) => {
   return result;
 };
 
+const AudioState = (wsMusic) => {
+  let playAudioIndex = [0, 0, 0, 0];
+  const [audioIndex, setAudioIndex] = useState(playAudioIndex);
+  const [audioPlaying, setAudioPlaying] = useState(false);
+
+  const audioToggle = () => {

```

Comment:
Let's think about capitalization. Do we also want to capitalize this function just like `Loading`?